docs/index.md

```diff
@@ -9,14 +9,80 @@ The [CLI](cli-reference/dub.md) can be used to
 - compile projects and external programs ([dub build](cli-reference/dub-build.md), [dub run](cli-reference/dub-run.md))
 - test projects ([dub test](cli-reference/dub-test.md))
 
-To see how to obtain dub, **go to the next page by pressing the button below**.
+DUB is bundled with most D compilers' distributions. However, it's also possible to install it separately.
+See [Installing DUB](getting-started/install.md) for details.
 
 <!-- old docs anchors for index page, all link to first steps -->
 <a id="own-projects"></a>
 <a id="adding-deps"></a>
 <a id="foreign-projects"></a>
 <a id="advanced-usage"></a>
 
-## Starting a new project
+## DUB Basics
 
-You can also skip ahead to [First steps](getting-started/first-steps.md) if you already have dub installed.
+DUB is a build tool similar to other modern languages build tools like Javascript's [npm](https://www.npmjs.com/)
+and Rust's [cargo](https://crates.io/).
+
+A file called `dub.sdl` (or `dub.json`) is used to configure a DUB project.
+
+> [SDL](https://sdlang.org/) is a "Simple Declarative Language" inspired by D's syntax.
+> Whether to use SDL or JSON for the DUB file is a [matter of taste](https://forum.dlang.org/thread/fehzcwpabruiyhpwiywj@forum.dlang.org).
+
+A DUB file may look like this:
+
+```sdl
+name "myproject"
+description "A minimal D application."
+authors "My Name"
+copyright "Copyright © 2024, My Name"
+license "Boost"
+
+dependency "libasync" version="~>0.9.5"
+
+configuration "library" {
+    targetPath "target/lib"
+}
+
+configuration "unittest" {
+    dependency "tested" version="~>0.9.5"
+	dependency "dshould" version="~>1.7.1"
```

Comment:
Fixed...

I didn't notice that because I am trying different editors (ended up with VS code as that's the only one that has worked well with D...). I set it up to show whitespace now.